draft-ietf-detnet-dp-sol-mpls-02.xml

```diff
@@ -120,8 +119,8 @@ LB: General comments:
 
   <abstract>
    <t>
-    This document specifies Deterministic Networking data plane encapsulation solutions. The
-    described data plane solutions is applied over an MPLS Packet Switched Networks.
+    This document specifies Deterministic Networking operation

```

Comment:
operation means also control and management plane. We cover data plane only here, with some hints to CP and MP.